deploy/scripts/mainnet/0017-CarbonVortex-upgrade.ts

```diff
@@ -0,0 +1,25 @@
+import { DeployFunction } from 'hardhat-deploy/types';
+import { HardhatRuntimeEnvironment } from 'hardhat/types';
+import { DeployedContracts, upgradeProxy, InstanceName, setDeploymentMetadata } from '../../../utils/Deploy';
+import { NATIVE_TOKEN_ADDRESS } from '../../../utils/Constants';
+
+/**
+ * upgrade carbon vortex 2.0 to v3:
+ * add maxInput to trade function
```

Comment:
description here can be 
* remove old vortex dependency

Reply:
done